### ISSUE TITLE
How to manually add Closed Captions (.SRT)

### ISSUE THREAD

**Issue by bigbluebutton-issue-import** [bigbluebutton/bigbluebutton#2501]
Originally reported on Google Code with ID 1749

```
I have been attempting to add Closed Captions to a test BBB environment that I have.
so far I have tried working with the "playback.js" file and the "AcornMediaPlayer"
javascript file. It appears that all the other URL parameters for Slides, Audio etc...
are dynamically specified in the javascript instead of statically in the "playback.html"
file. I certainly think the current setup makes more sense, I just need help sorting
out where I need to link to my .SRT files. 

To summarize, I just need to know where/ how to link out to the CC files (.SRT's).
```

Reported by `ctrlaltdelete63` on 2014-04-17 22:16:57
